onnxruntime/test/python/quantization/op_test_utils.py

```diff
@@ -36,7 +35,7 @@ def InputFeedsNegOneZeroOne(n, name2shape):
 
 
 def check_op_type_order(testcase, model_to_check, ops):
-    if isinstance(model_to_check, string_types):
+    if isinstance(model_to_check, str):
         model = onnx.load(model_to_check)
     elif isinstance(model_to_check, onnx.ModelProto):
         model = model_to_check
```

Comment:
op_test_utils.py has been updated recently and there is one more place using string_types at Line 81. Updating that line should fix current CI failures. Thanks!